### DISCUSSION TITLE
The billing support took too long to respond

### DISCUSSION THREAD

**Discussion by brieferid** [community/community#112390] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have an urgent issue with my billing and have created a ticket. However, it has been 1 day and I have not received any response yet. Is this normal?

**Comment by stjepanDeveloper** [comment]
Hello @brieferid , 
This depends on the type of account/subscription you have. Github premium support usually takes half an hour for urgent and 4 hours for high urgency tickets.
Gihub free account tickets can vary in response time from couple of days to couple of weeks. They vary a lot in response time. So it's not strange your ticket is taking this long. 
If you're on a paid plan then you should definetly check again with them.
